src/multi-input/index.jsx

```diff
@@ -1,48 +1,66 @@
+import { v4 as uuid } from 'uuid';
 import React, { useState, useEffect } from 'react';
 import { Button, Input } from '@ukhomeoffice/react-components';
+import castArray from 'lodash/castArray';
 
-function Item({ value, onRemove, showRemove, onChange }) {
+function Item({ item, onRemove, showRemove, onChange, name, disabled }) {
+  const isDisabled = disabled.includes(item.id);
   return (
     <div className="multi-input-item">
-      <Input type="text" value={value} onChange={onChange} label="" />
+      <Input type="text" value={item.value} onChange={onChange} label="" name={name} disabled={isDisabled} />
```

Comment:
I think because `Input` defaults the id to the `name` prop then all of the input elements will end up with the same id.
```suggestion
      <Input type="text" value={item.value} onChange={onChange} label="" name={name} id={item.id} disabled={isDisabled} />
```

Reply:
orly? ok let's go with that then 👍